pom.xml

```diff
@@ -431,6 +433,12 @@
             <version>3.2.1</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.openjdk.jol</groupId>
+            <artifactId>jol-core</artifactId>
+            <version>0.16</version>
+        </dependency>
+
```

Comment:
Please make sure security is ok with this dependency

Reply:
I checked the dependency list and they dont have any dependencies except junit. I will still do the security review.